packages/forms/src/patterns/select-dropdown.ts

```diff
@@ -0,0 +1,121 @@
+import * as z from 'zod';
+
+import { type SelectDropdownProps } from '../components.js';
+import {
+  type Pattern,
+  type PatternConfig,
+  validatePattern,
+} from '../pattern.js';
+import { getFormSessionValue } from '../session.js';
+import { safeZodParseFormErrors } from '../util/zod.js';
+
+const configSchema = z.object({
+  label: z.string().min(1),
+  required: z.boolean(),
+  options: z
+    .object({
+      value: z
+        .string()
+        .regex(/^[A-Za-z][A-Za-z0-9\-_:.]*$/, 'Invalid Option Value'),
+      label: z.string().min(1),
+    })
+    .array(),
+});
+export type SelectDropdownPattern = Pattern<z.infer<typeof configSchema>>;
+
+const PatternOutput = z.string();
+type PatternOutput = z.infer<typeof PatternOutput>;
+
+export const selectDropdownConfig: PatternConfig<
+  SelectDropdownPattern,
+  PatternOutput
+> = {
+  displayName: 'Select Dropdown',
+  iconPath: 'dropdown-icon.svg',
+  initial: {
+    label: 'Select-dropdown-label',
+    required: true,
+    options: [
+      { value: 'value1', label: '-Select-' },
```

Comment:
If this default is selected, does the pattern accept it? Should it be an empty value?

I think I'd vote for always rendering a default option via the template, and omitting it from the config data definition. For example, how it's handled in the [address pattern component](https://github.com/GSA-TTS/atj-platform/blob/main/packages/design/src/Form/components/Address/index.tsx#L97). If the field is required, the pattern can then treat the empty string as an error.

Reply:
I totally agree with you! I initially implemented it as part of template then changed it to current version thinking that it potentially be a dynamic value like "select", "select a country", "select a month" etc. However, I think it would be better to keep it as generalized value ("- select -") for all scenarios and render it through the template. Will update it!